main.tex

```diff
@@ -18,6 +18,49 @@
 \begin{document}
 
 \maketitle
+% 2.3 in der Gliederung
+\subsection{Start in ausgewählten Regionen}
 
+Im Anschluss an den formalen Start fingen erneute Debatten um "Net-Neutrality" an Beoden zu gewinnen. Facebooks Dienst sei nicht konform mit dem Bestreben eines freien Internets, trotz der Bewerbung des Produkts als "kostenloses Facebook"\cite[3]{prasad2017}
+ , später als ein "step towards digital equality".
+
+\medskip
+
+In der ländlichen Bevölkerung gab es auch geteilte Meinungen hinsichtlich des Zugangs zum Internet, zumindest bei jenen, die sich selbst informiert haben oder (von zum Beispiel Nachrichtendiensten wie "Digit.in") \textcite{digitYT}
+informiert werden. Hier kommt es jdeoch zumeist zur Ignoranz des großen Ganzen, der Dienst wird als positiv entgegengenommen und genutzt, Aktionen wie die wortwörtlich vorgeschriebene Unterschreibung zugunsten von Facebook in der Debatte um Netzneutralität \textcite{ndtvYT}
+werden ohne weiteres Hinterfragen und potenziell aufgrund von Unwissen unterstützt. Einzig ein initielles Interesse an einem kostenlosen Dienst im allgemeinen Sinne ist gemein.
+
+\medskip
+
+Anzumerken bei alledem ist, dass der Großteil der Bevölkerungsgruppe, die in Facebooks Werbekampanien als Zielgruppe des Dienstes beschrieben werden, entweder keinen oder nur einen gerinfügigen Nutzen genießt. Jene, die sich den Besitz eines digitalen Endgeräts (id est Smartphones, etc.) leisten können, sind aufgrund der geringen Preise auch in der Lage, eine stabilere und schnellere Internetverbindung als Facebooks Dienst sie anbietet zu erwerben. Der Rest, also Personen, die (zu Teilen weit) unterhalb der Armutsgrenze leben, haben weder Wege, FreeBasics zu nutzen, noch einen Grund dafür. \cite[257]{everydayLife}
```

Comment:
> [...] werden, entweder keinen oder nur einen [...]

[...] werden, entweder keinen *,* oder nur einen [...]

Reply:
> [...] in der Lage, eine stabilere und schnellere Internetverbindung als Facebooks Dienst sie anbietet zu erwerben.

[...] in der Lage, eine stabilere und schnellere Internetverbindung als Facebooks Dienst sie anbietet, zu erwerben.